libs/features/mas/web-components/src/hydrate.js

```diff
@@ -12,29 +12,33 @@ export async function hydrate(fragmentData, merchCard) {
     if (!variant) return;
     fragment.model = fragment.model;
 
-    merchCard.variantLayout?.refs?.forEach((ref) => ref.remove());
+    // remove all previous slotted content except the default slot
+    merchCard.querySelectorAll('[slot]').forEach((el) => {
```

Comment:
this is handle use-case when card is refreshed?
how about we have a method inside merch-card to do it? We might need to also remove all the previous attributes

Reply:
since `merch-card` is intended to be modular, I think keeping this fragment related logic in hydrate.js ok.
Also, for Milo cards, it doesn't really make sense as we don't support refetching the content.